content/en/docs/concepts/storage/ephemeral-volumes.md

```diff
@@ -0,0 +1,263 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- msau42
+- xing-yang
+- pohly
+title: Ephemeral Volumes
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+This document describes _ephemeral volumes_ in Kubernetes. Familiarity
+with [volumes](/docs/concepts/storage/volumes/) is suggested, in
+particular `PersistentVolumeClaim` and `PersistentVolume`.
+
+<!-- body -->
+
+Some application need additional storage but don't care whether that
+data is stored persistently across restarts. For example, caching
+services are often limited by memory size and can move infrequently
+used data into storage that is slower than memory with little impact
+on overall performance.
+
+Other applications expect some read-only input data to be present in
+files, like configuration data or secret keys.
+
+_Ephemeral volumes_ are designed for these use cases. Because volumes
```

Comment:
I think it's important to mention somehwere in the intro that ephemeral volumes follow the pod's lifetime and get created and deleted along with the pod.

Reply:
Replaced "created anew for each pod" with "follow the pod's lifetime and get created and deleted along with the pod".

Or rather, *Pod* (capital case)... I also fixed that elsewhere.